integration-tests-jvm/opensearch/src/main/java/org/apache/camel/quarkus/component/opensearch/it/OpensearchRoutes.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.opensearch.it;
+
+import org.apache.camel.builder.RouteBuilder;
+
+public class OpensearchRoutes extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+        from("direct:indexDoc")
+                .toD("opensearch://opensearch?operation=Index");
```

Comment:
Nitpick - I guess we don't need `toD` for anything here. I don't see any dynamic URIs being used to we can just use the standard `.to(..)` for everything.